public/editor/scripts/project-list.js

```diff
@@ -119,6 +122,81 @@ require(["jquery", "constants", "analytics", "moment"], function($, Constants, a
         project.remove();
       }
     });
+  }
+
+  function onDeleteError(err) {
+    err = err || new Error("unknown network error");
+    console.error(err);
+  }
+
+  $(".delete-button").click(function() {
+    // TODO: we can do better than this, but let's at least make it harder to lose data.
+    if(!window.confirm("{{ deleteProjectConfirmationText }}")) {
+      return false;
+    }
+        
+    while(projectsToDelete.length > 0) {
+      deleteProject($(projectsToDelete.shift()));
+    }
+
+    $(".delete-button").toggle();
+  });
+
+  function findProject(project){
+    for(var i = 0; i < projectsToDelete.length; i++){
+      if(projectsToDelete[i].attr("id") === project.attr("id")) {
+        return i;
+      }
+    }
+    return -1;
+  }
+
+  $(".project-delete").click(function() {
+    if($(".delete-button").css("display") === "none") {
+      $(".delete-button").toggle();
+    }
+
+    var project = $(this).closest(".project");
+    var projectSelector = "#" + project.attr("id");
+    var deleteIndex = findProject(project);
+    var disabledLink = {"cursor":"default", "text-decoration":"none"};
+    var enabledLink = {"cursor":"pointer", "text-decoration":"underline"};
+    
+    if(deleteIndex === -1) {
+      projectsToDelete.push(project);
+      $(projectSelector + " .edit-link").on("click", function(link) {
+        link.preventDefault();
+      }).css({"cursor":"default"});
+
+      if($(projectSelector + " .published-link").length !== 0) {
+        $(projectSelector + " .published-link").on("click", function(link) {
+          link.preventDefault();
+        }).css(disabledLink);
+        $(projectSelector + " .remix-link").on("click", function(link) {
+          link.preventDefault();
+        }).css(disabledLink);
+      }
+
+      project.css("background", "rgba(255,0,0,.08)");
```

Comment:
Please apply a class to this element and change the background with a CSS rule instead of applying styles directly in JavaScript